onebusaway-android/build.gradle

```diff
@@ -27,11 +27,14 @@ apply plugin: 'com.android.application'
 
 repositories {
     jcenter()
-    mavenLocal()
     maven {
         // OBA Releases - for comparator to sort alphanumeric routes
         url "http://nexus.onebusaway.org/nexus/content/repositories/releases"
```

Comment:
Yeah, this is a good improvement - it prevents the developer from having to do the mvn install themselves and simplifies the build setup.  It may result in performance improvements in Android Studio too - I recall a few issues where Android Studio indexes the entire local maven repo which results in huge files that Android Studio then has to parse.